packages/platform-sdk-eth/src/services/transaction.ts

```diff
@@ -204,7 +204,7 @@ export class TransactionService implements Contracts.TransactionService {
 						},
 					],
 					payable: false,
-					stateMutability: "nonpayable",
+					stateMutpermissions: "nonpayable",
```

Comment:
`stateMutpermissions` => `stateMutPermissions`

Reply:
I would even go with stateMutabilityPermissions

Reply:
That's a parameter defined by the web3 package, not us.